api/src/main/resources/application.yml

```diff
@@ -94,13 +99,24 @@ csm:
       reader:
         username: "storage_reader_username"
         password: "storage_reader_password"
+      writer:
+        username: "storage_writer_username"
+        password: "storage_writer_password"
       admin:
         username: "storage_admin_username"
         password: "storage_admin_password"
     eventbus:
       host: "localhost"
-      username: "eventbus_admin_username"
-      password: "eventbus_admin_password"
+      port: "5672"
+      default-exchange: "csm-exchange"
+      default-queue: "csm"
+      default-routing-key: "csm"
```

Comment:
Same remark as in the common repo, this is not something currently configured on the deployment side.

Reply:
This is part of current Cosmotech RabbitMqConfig, will see next week to:

- remove defaultQueue creation
- change defaultExchange to amq.default (which is here as default when rabbitMQ is deployed)
- remove defaultRoutingKey